publishable/database/seeds/DataRowsTableSeeder.php

```diff
@@ -166,7 +166,7 @@ public function run()
                 'edit'         => 1,
                 'add'          => 1,
                 'delete'       => 1,
-                'details'      => '',
+                'details'      => '{"slugify": {"origin": "title", "forceUpdate": true}}',
```

Comment:
Can you update it to be ```
json_encode([
    'slugify' => [
        'origin' => 'title',
        'forceUpdate' => true,
    ],
]),
```

Might be a little cleaner, I will cleanup the other rows.